gr-uhd/apps/uhd_fft

```diff
@@ -431,7 +432,7 @@ class uhd_fft(gr.top_block, Qt.QWidget, UHDApp):
         self.qtgui_waterfall_sink_x_0.set_frequency_range(self.freq, self.samp_rate)
         self.usrp.set_samp_rate(self.samp_rate)
         for c in range(len(self.channels)):
-            self.usrp.set_bandwidth(self.samp_rate, c)
+            self.usrp.set_bandwidth(self.samp_rate + abs(self.lo_offset), c)
```

Comment:
I realized we need abs() after making the original PR, so I added a second commit with abs(self.lo_offset). Do you see this line on your review? It shows up for me like this: 

`self.usrp.set_bandwidth(self.samp_rate + abs(self.lo_offset), c)`

If you dont see that, I'll try to find why the second commit doesnt show up in the PR. If you prefer, I can rebase and redo the PR... up to you.